pages/blogs/home.ftd

```diff
@@ -0,0 +1,96 @@
+-- import: ui.fifthtry.com/components/page as p
```

Comment:
These files can only contain structure/components, not content. Move the content to scenario files.